src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -86,13 +88,19 @@ public void initDefaultCommand() {
 
     public Command driveNormal() {
         return new Command("driveNormal", this) {
+            private double triggerSpeed = 0;
+            private double thumbstickSpeed = 0;
 
             @Override
             protected void execute() {
-                drive.arcadeDrive(
-                        +Robot.driveController.getTriggerAxis(Hand.kRight)
-                                - Robot.driveController.getTriggerAxis(Hand.kLeft),
-                        Robot.driveController.getX(Hand.kLeft));
+                triggerSpeed = -Robot.driveController.getTriggerAxis(Hand.kRight)
+                        + Robot.driveController.getTriggerAxis(Hand.kLeft);
```

Comment:
Reminder to myself to put the clamp function in chopshoplib because this is ugly

Reply:
Resolving, as I've added clamp to chopshoplib (we can use it later on)